doc/tutorials/ios-app.md

```diff
@@ -270,30 +277,95 @@ output will appear similar to the following:
 ```bash
 INFO: Found 1 target...
 Target //ios-app:ios-app up-to-date:
-  bazel-bin/ios-app/ios-app.ipa
-INFO: Elapsed time: 0.565s, Critical Path: 0.44s
+  bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa
+INFO: Elapsed time: 0.141s, Critical Path: 0.00s
 ```
 
 ### Find the build outputs
 
 The `.ipa` file and other outputs are located in the
-`$WORKSPACE/bazel-bin/ios-app` directory.
+`$WORKSPACE/bazel-out/applebin_ios-ios_sim_arm64-fastbuild-ST-4e6c2a19403f/bin/ios-app/ios-app.ipa` directory.
+
+### Build the app in the simulator
+
+`rules_apple` supports running an app directly in the iOS Simulator.
+Replace `build` with `run` in the previous command to both build and
+run the application:
+
+```bash
+bazel run //ios-app:ios-app
+```
+
+Note: `--ios_simulator_device` and `--ios_simulator_version` control which
+version and device will be used when launching the app.
+
+### Generate an Xcode project
+
+There are a few community-provided solutions (such as [rules_xcodeproj](https://github.com/buildbuddy-io/rules_xcodeproj)
+and [Tulsi](https://github.com/bazelbuild/tulsi)) to help generating Xcode 
```

Comment:
Since Tulsi has a website, should this link there instead of the GitHub repo?

```suggestion
and [Tulsi](https://tulsi.bazel.build/)) to help generating Xcode 
```